_includes/how-to-guides/deployment/access-your-code.md

```diff
@@ -1,3 +1,25 @@
+## Private GitHub repositories
+
+Cloud 66 offers a native integration with GitHub. When creating a new application, click on the *Deploy from Github* link and we will install the **Cloud 66 Deployments** GitHub application on your account. This gives us read-only access to the repositories you specify. 
+
+### Adding or updating GitHub accounts
+
+If you have access to multiple GitHub accounts, you will need to add each account separately. To  add more accounts:
+
+1. Log into your Cloud 66 Dashboard
+2. Click on your avatar (top right) and then *Account Settings*
+3. Click on *External Services* in the **Settings** panel (on the left)
+4. Click the *Configure GitHub app* link next to the GitHub app
+5. Click on the name of the account to add - you will be sent to GitHub to configure your access
```

Comment:
This is selecting a github account - I think? You are already in github at this point. Seems a bit confusing

Reply:
V good point. I have updated it, thanks B!